src/pages/projectSettings/tabs/NotificationsTab/getFormSchema.tsx

```diff
@@ -58,6 +63,53 @@ export const getFormSchema = (
             },
           },
         },
+        ...(projectType !== ProjectType.Repo && {
+          banner: {
+            type: "object" as "object",
+            title: "",
+            properties: {
+              bannerData: {
+                type: "object" as "object",
+                title: "Project Banner",
+                description:
+                  "Add a banner to pages that represent data from this project.",
```

Comment:
Should we mention here that including JIRA ticket numbers will create links to JIRA?